### ISSUE TITLE
Release 1.1.2 breaks Android build

### ISSUE THREAD

**Issue by AlexanderEggers** [DataDog/dd-sdk-reactnative#296]
Thanks for taking the time for reporting an issue!

**Describe what happened**

When running the latest release, the android build breaks with the following build error:
```
e: /Users/alex/Documents/Projects/myapp/node_modules/@datadog/mobile-react-native/android/src/main/kotlin/com/datadog/reactnative/ProxyAuthenticator.kt: (21, 43): Using 'code(): Int' is an error. moved to val
e: /Users/alex/Documents/Projects/myapp/node_modules/@datadog/mobile-react-native/android/src/main/kotlin/com/datadog/reactnative/ProxyAuthenticator.kt: (26, 54): Using 'code(): Int' is an error. moved to val
e: /Users/alex/Documents/Projects/myapp/node_modules/@datadog/mobile-react-native/android/src/main/kotlin/com/datadog/reactnative/ProxyAuthenticator.kt: (34, 36): Using 'scheme(): String' is an error. moved to val
e: /Users/alex/Documents/Projects/myapp/node_modules/@datadog/mobile-react-native/android/src/main/kotlin/com/datadog/reactnative/ProxyAuthenticator.kt: (41, 31): Using 'charset(): Charset' is an error. moved to val
e: /Users/alex/Documents/Projects/myapp/node_modules/@datadog/mobile-react-native/android/src/main/kotlin/com/datadog/reactnative/ProxyAuthenticator.kt: (43, 33): Using 'request(): Request' is an error. moved to val
```

**Comment by HannahCarney** [comment]
Sorry, this is probably my fault for raising bug: https://github.com/DataDog/dd-sdk-reactnative/issues/288

We are running on old version of react native (DD says it supports my version) and gradle 6.9 therefore also on a different version of okhttp. Have a feeling this is related to okhttp, and sorry my bug and their quick fix broke your build!!

**Comment by louiszawadzki** [comment]
Hi,

The issue has been fixed in version `1.1.3`. 
It also solves the issue for older versions of React Native using okhttp version 3.

I'm closing the issue for now,  feel free to reopen it if you encounter any further problem.
